proxy/http/HttpTransact.cc

```diff
@@ -5723,9 +5723,13 @@ HttpTransact::initialize_state_variables_from_request(State *s, HTTPHdr *obsolet
     s->cache_info.action = CACHE_DO_NO_ACTION;
   }
 
+  bool do_increment_stat = (0 == s->method);
```

Comment:
So without this check unset methods count towards tunnel.   Won't this remove some legitimate tunnel counts like tr_pass tunnels?  In what cases are we seeing unset methods?

Reply:
OK done.